scipy/stats/tests/test_morestats.py

```diff
@@ -1200,6 +1200,88 @@ def test_empty(self):
         assert_(stats.boxcox_normplot([], 0, 1).size == 0)
 
 
+class TestYeojohnson_llf(object):
+
+    def test_array_like(self):
+        np.random.seed(54321)
+        x = stats.norm.rvs(size=100, loc=10)
+        lmbda = 1
+        llf = stats.yeojohnson_llf(lmbda, x)
+        llf2 = stats.yeojohnson_llf(lmbda, list(x))
+        assert_allclose(llf, llf2, rtol=1e-12)
+
+    def test_empty(self):
+        assert_(np.isnan(stats.yeojohnson_llf(1, [])))
+
+
+class TestYeojohnson(object):
+
+    def test_fixed_lmbda(self):
```

Comment:
can you add further test cases for negative x and `lmbda == 2` / `lmbda != 2`?

and also vector x with negative and positive numbers?

Reply:
Just did in 1c898bb